compose/ui/ui-unit/src/commonMain/kotlin/androidx/compose/ui/unit/Density.kt

```diff
@@ -159,4 +160,35 @@ interface Density {
     } else {
         DpSize.Unspecified
     }
+
+    /**
+     * Convert a [Offset] to a [DpOffset].
+     */
+    @Stable
+    fun Offset.toDpOffset(): DpOffset = if (isSpecified) {
```

Comment:
This adds a new public API to common source set. Unless we upstream this change, users would have errors in android when these methods are used in common code. 

Maybe we can move this code to skikoMain (or even jsNativeMain)? 
We can't have multiple extension receivers yet, so it can look like `fun Density.offsetToDpOffset(offset: Offset): DpOffset`?

Reply:
Thanks.
I changed it, but the signature now looks a little bit different:
```Kotlin
fun Offset.toDpOffset(density: Density): DpOffset?
```